src/main/java/org/broadinstitute/hellbender/tools/AddOriginalAlignmentTags.java

```diff
@@ -19,10 +21,12 @@
         programGroup = ReadDataManipulationProgramGroup.class
 )
 @ExperimentalFeature
+@WorkflowProperties
```

Comment:
What did we end up going with for the default memory usage?

Reply:
Its 4G (IIRC, I had previously incorrectly said that it was 1G, but it was and still is 4G). We still might want to find away to customize that for Picard.